src/psyclone/core/signature.py

```diff
@@ -107,6 +108,10 @@ def __str__(self):
 
     # ------------------------------------------------------------------------
     def to_language(self, component_indices, language_writer=None):
```

Comment:
I'd prefer not, since I actually want to get rid of `to_language` (once we don't have strings in the PSyIR anymore). The __str__ is then mostly used for testing (easier to understand what a test means), and debugging. In these cases, it does not really matter what language it is (e.g. in `    assert str(sig_c_d_b_a_b) == "c%d%b%a%b"`).
`)